e2e_playwright/conftest.py

```diff
@@ -248,6 +248,32 @@ def app(page: Page, app_port: int) -> Page:
     return page
 
 
+@pytest.fixture(scope="function")
+def app_with_params(page: Page, app_port: int, request) -> Tuple[Page, Dict]:
```

Comment:
* probably worth naming this `app_with_query_params` to be more specific
* is `request` of type `FixtureRequest`?